raddb/mods-config/perl/example.pl

```diff
@@ -49,10 +49,8 @@
 
 # Multi-value attributes are mapped to perl arrayrefs.
 #
-#  update request {
-#    Filter-Id := 'foo'
-#    Filter-Id += 'bar'
-#  }
+#  &request.Filter-Id := 'foo'
+#  &request.Filter-Id += 'bar'
```

Comment:
The should be:

```
&request.Filter-Id := 'foo'
&request += {
    &request.Filter-Id += 'bar'
}
```

It's a little weird, but `+=` is now _always "concatenate strings".